webapp/app/js/perf.js

```diff
@@ -179,6 +179,83 @@ perf.controller('PerfCtrl', [ '$state', '$stateParams', '$scope', '$rootScope',
       });
     }
 
+    function plotoverviewGraph() {
+      $scope.seriesList.forEach(function(series) {
+        series.flotSeries.points.show = false;
+        series.flotSeries.lines.show = false;
+        if (series.visible){
+          series.flotSeries.lines.show = true;
+        }
```

Comment:
You could replace this if clause (and the line above it) with "series.flotSeries.lines.show = series.visible"